experimental/dds/sequence-deprecated/package.json

```diff
@@ -0,0 +1,108 @@
+  "scripts": {
+    "build": "npm run build:genver && concurrently npm:build:compile npm:lint && npm run build:docs",
+    "build:commonjs": "npm run tsc && npm run typetests:gen && npm run build:test",
+    "build:compile": "concurrently npm:build:commonjs npm:build:esnext",
+    "build:docs": "api-extractor run --local --typescript-compiler-folder ../../../node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
+    "build:esnext": "tsc --project ./tsconfig.esnext.json",
+    "build:full": "npm run build",
+    "build:full:compile": "npm run build:compile",
+    "build:genver": "gen-version",
+    "build:test": "tsc --project ./src/test/tsconfig.json",
+    "ci:build:docs": "api-extractor run --typescript-compiler-folder ../../../node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/* ../../../_api-extractor-temp/",
+    "clean": "rimraf dist lib *.tsbuildinfo *.build.log",
+    "eslint": "eslint --format stylish src",
+    "eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+    "lint": "npm run eslint",
+    "lint:fix": "npm run eslint:fix",
+    "test": "npm run test:mocha",
+    "test:coverage": "nyc npm test -- --reporter xunit --reporter-option output=nyc/junit-report.xml",
+    "test:mocha": "mocha --recursive dist/test/**/*.spec.js -r node_modules/@fluidframework/mocha-test-setup --unhandled-rejections=strict",
+    "test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
+    "test:newsnapfiles": "node dist/test/createSnapshotFiles.js",
+    "testfarm": "node dist/test/testFarm.js",
+    "tsc": "tsc",
+    "tsfmt": "tsfmt --verify",
+    "tsfmt:fix": "tsfmt --replace",
+    "typetests:gen": "fluid-type-validator -g -d ."
+  },
+  "nyc": {
+    "all": true,
+    "cache-dir": "nyc/.cache",
+    "exclude": [
+      "src/test/**/*.ts",
+      "dist/test/**/*.js"
+    ],
+    "exclude-after-remap": false,
+    "include": [
+      "src/**/*.ts",
+      "dist/**/*.js"
+    ],
+    "report-dir": "nyc/report",
+    "reporter": [
+      "cobertura",
+      "html",
+      "text"
+    ],
+    "temp-directory": "nyc/.nyc_output"
+  },
+  "dependencies": {
+    "@fluidframework/common-definitions": "^0.20.1",
+    "@fluidframework/common-utils": "^0.32.1",
+    "@fluidframework/container-utils": "^2.0.0",
+    "@fluidframework/core-interfaces": "^2.0.0",
+    "@fluidframework/datastore-definitions": "^2.0.0",
+    "@fluidframework/default-map": "^2.0.0",
+    "@fluidframework/merge-tree": "^2.0.0",
+    "@fluidframework/protocol-definitions": "^0.1029.1000-0",
+    "@fluidframework/runtime-definitions": "^2.0.0",
+    "@fluidframework/runtime-utils": "^2.0.0",
+    "@fluidframework/sequence": "^2.0.0",
+    "@fluidframework/shared-object-base": "^2.0.0",
+    "@fluidframework/telemetry-utils": "^2.0.0",
+    "uuid": "^8.3.1"
+  },
+  "devDependencies": {
+    "@fluid-internal/stochastic-test-utils": "^2.0.0",
```

Comment:
probably worth doing a pass over devDeps and deps--I don't think this package will be referenced by any moved code.